std/string.d

```diff
@@ -62,19 +80,55 @@ strings, use $(D char[]), $(D wchar[]), or $(D dchar[]). The $(D xxxstring)
 types are preferable because they don't exhibit undesired aliasing, thus
 making code more robust.
 
+The following functions are publicly imported:
+
+$(BOOKTABLE ,
+$(TR $(TH Module) $(TH Functions) )
+    $(TR $(TD algorithm)
+        $(TD
+		 $(XREF algorithm, cmp)
+		 $(XREF algorithm, count)
+		 $(XREF algorithm, endsWith)
+		 $(XREF algorithm, startsWith)
+    ))
+    $(TR $(TD array)
+        $(TD
+		 $(XREF array, join)
+		 $(XREF array, split)
+    ))
+    $(TR $(TD format)
+        $(TD
+		 $(XREF format, format)
+		 $(XREF format, sformat)
+    ))
+    $(TR $(TD uni)
+        $(TD
+		 $(XREF uni, icmp)
+		 $(XREF uni, toLower)
+		 $(XREF uni, toLowerInPlace)
+		 $(XREF uni, toUpper)
+		 $(XREF uni, toUpperInPlace)
+    ))
+)
+
 Not all functions for D _string handling are in this module. Functions related
```

Comment:
Does it? I meant to go from local to global: local functions, publicly imported functions, other modules. Assuming someone that looks into the string module might want to know what is there first.

I am happy to change this, of course.